pontoon-to-webext.js

```diff
@@ -0,0 +1,164 @@
+#! /usr/bin/env node
+
+/* eslint-disable promise/avoid-new */
+
+const propertiesParser = require('properties-parser');
+const path = require('path');
+const FS = require('q-io/fs');
+const argv = require('minimist')(process.argv.slice(2));
+
+const Habitat = require('habitat');
```

Comment:
Eh, okay, thought maybe it was a leftover from other tinkering that got deleted. Because I do that all the time. :man_shrugging: